doc/datastorage_setup

```diff
@@ -0,0 +1,36 @@
+Installing ZeroMQ:
+
+# run in sudo
+# Before installing, make sure you have installed all the needed packages
+1.sudo apt-get install libtool pkg-config build-essential autoconf automake
+
```

Comment:
Is this an intentional blank line? If not, please remove it.

Reply:
Fixed